apps/admin_app/lib/admin_app/order/order.ex

```diff
@@ -65,22 +65,15 @@ defmodule AdminApp.OrderContext do
     end)
   end
 
-  def order_list("complete") do
-    query =
-      from(
-        order in Order,
-        where: order.state == "complete"
-      )
-
-    Repo.all(query)
+  def order_list("complete", rummage) do
+    {queryable, _rummage} = Order.rummage(rummage)
+    query = from(p in queryable, where: p.state == "complete")
+    Repo.all(query) |> Repo.preload([:user, :packages, [line_items: :product]])
```

Comment:
use `load_order/1` functions it's created for running the query and preloading associations.